content/blog/2017/08/2017-08-05-remoting-update.adoc

```diff
@@ -0,0 +1,129 @@
+* Jenkins shows an administrative warning when obsolete protocols are enabled
+
+### Compatibility notes
+
+Old instances won't be affected, protocols will be still enabled for them. 
+Newly installed instances launched with the  `-Djenkins.install.runSetupWizard=false` flag (e.g. configuration-as-code) won't be affected as well, 
+the initial configuration is being managed by admins there.
+
+"New" Jenkins instances installed via setup wizard **may be affected** in edge cases. Examples:
+
+* Agents with Remoting older than 3.0 will be unable to connect.
+** **Mitigation:** Before updating make sure Remoting is not bundled custom Docker images, AMIs, etc.
+* Swarm Plugin: old versions of Swarm Client (before 3.3) will be unable to connect to Jenkins, because Remoting 2.x is bundled
+** **Mitigation**: Update Swarm Client
+* **Very** old jenkins-cli.jar without CLI2 support will be unable to connect. 
+** **Mitigation**: Do not use Remoting-based CLI on new instances (see link:/blog/2017/04/11/new-cli/[this blogpost])
+
+## Upgrade to Java 8
+
+Starting from 2.54 Jenkins requires Java 8 to run 
+(link:https://jenkins.io/blog/2017/04/10/jenkins-has-upgraded-to-java-8/[announcement blog post]).
+This version is also required for Jenkins LTS 2.60.1.
+
+Remoting was staying on Java 7 for a while for backporting purposes, 
+but it will be also upgraded to Java 8 in the Remoting 3.11 release.
+This change is tracked as link:https://issues.jenkins-ci.org/browse/JENKINS-43985[JENKINS-43985] in Jenkins JIRA.
+
+### Compatibility notes
+
+The update does not cause compatibility issues in common use-cases.
+However, there may be issues in custom Jenkins core builds and packaging.
+There are several examples below.
+
+* Jenkins instances with built-in Remoting versions will **NOT** be affected, Java 8 is already required there
+* Users of community-provided Docker packages (link:https://github.com/jenkinsci/docker-slave[docker-slave], 
+  link:https://github.com/jenkinsci/docker-jnlp-slave[docker-jnlp-slave]) will **NOT** be affected, 
+Java 8 is already required there
+* **Custom** Jenkins WAR file builds targeting Java 7 may fail to build/run if they bundle Remoting 3.11 or later
+* **Custom** Jenkins agent instances (manually installed hosts, VM snapshots, Docker packages, AMIs, etc.) may fail if they download the latest Remoting version and use Java 7 
+
+### Java 9 support
+
+One may ask if Java 9 is supported in Remoting.
+As Jenkins core, right now in Remoting this Java version is **not supported** and **not tested**.
+It may work in some configurations, but itis not guaranteed.
+
+As a consequence, it is not recommended to run Remoting with Java 9 right now. 
+It is also not recommended to use link:https://plugins.jenkins.io/maven-plugin[Maven Integration Plugin] to run builds on Java 9.
+
+## What's next?
+
+There are some ongoing activities on Remoting fronts. 
+They are grouped to several EPICs:
```

Comment:
s/EPICs/Epics/

EPIC is not an acronym, it's a proper noun. Please replace in the bullet point list below as well